src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -151,13 +154,16 @@ export default function MeasureInformation(props: MeasureInformationProps) {
   );
 
   const handleEndorserChange = (selectedValue: string) => {
+    if (selectedValue === "" || selectedValue === "-") {
+      selectedValue = null;
+    }
```

Comment:
@mcmcphillips Is the `-` needed for 508 reasons?

Reply:
@jkotanchik-SB I think it was from the figma design.

Reply:
would it be more appropriate to use the `placeholder` prop instead of setting the actual value to `-`?

Reply:
looks like there already is a `placeholder` for endorser set to `-` so I'm curious how/why we're actually setting a formik value to `-`

Reply:
That was one of the values that was requested to be used on the backend in another story. It's expecting `-`, `NQF`, or `null`. I've realized that may have been what was requested in the ticket, but probably isn't actually what was wanted in reality.